app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class UserController extends Controller
+{
+    //
+    public function __construct()
+    {
+
+    }
+    public function showErrors()
+    {
+        return 'Con xin lỗi chú';
```

Comment:
????? what the hell ??

Reply:
con cac dap chet me may gio thang leader l